amlb/job.py

```diff
@@ -64,7 +65,10 @@ def is_state_transition_ok(cls, old_state: State, new_state: State):
 
     printer = pprint.PrettyPrinter(indent=2, compact=True)
 
-    def __init__(self, name="", timeout_secs=None, priority=None, raise_on_failure=False):
+    def __init__(self, name: str = "",
+                 timeout_secs: int = -1,
+                 priority: int = None,
```

Comment:
```suggestion
                 priority: Optional[int] = None,
```
since `None`  is a valid option, would also need to add `Optional` in imports.